Doc: Updating vtex-io-documentation-running-native-ab-testing.md (#903)
vtexdocs/dev-portal-content: docs/vtex-io/App-Guides/ab-tests/vtex-io-documentation-running-native-ab-testing.md

| **Expected Loss**            | Anticipated percentage of conversion loss for the store if the lower conversion rate workspace is selected as the winner (based on **Conversion** results). |
| **N. of Sessions**           | Total number of sessions for each workspace since the beginning of the test.                        |
| **N. of Sessions (last 24hrs)** | Number of sessions for each workspace in the past 24 hours during the test.                           |

> | **Revenue**             | Total value of orders placed by users navigating in the version of your store associated to each workspace since the beginning of the test (according to the currency configured in the store).|

Reviewer: ```suggestion
| **Revenue**             | The total value of orders placed by users navigating the version of your store associated with each workspace since the beginning of the test, according to the currency configured within the store. |
```

> | **Revenue**             | Total value of orders placed by users navigating in the version of your store associated to each workspace since the beginning of the test (according to the currency configured in the store).|

Reviewer 2: ```suggestion
| **Revenue**             | Total value of orders placed by users navigating the store version associated with each workspace since the beginning of the test (expressed in the store's native currency).|
```

Reviewer 2: - Sugestão para deixar a descrição mais concisa.
- "associated to" é aceitável, mas o mais correto é ["associated with"](https://www.oxfordlearnersdictionaries.com/us/definition/english/associated) segundo os dicionários.
- Sugestão para utilizar "expressed in" ao invés de "according to" para deixar mais explícito que estamos falando sobre como essa unidade é expressa

> | **Revenue (last 24hs)** | Value of orders placed by users navigating in the version of your store associated to each workspace in the past 24 hours durint the test (according to the currency configured in the store).|

Reviewer: ```suggestion
| **Revenue (last 24hs)** | The value of orders placed by users navigating the version of your store associated with each workspace in the past 24 hours during the test, according to the currency configured within the store. |
```

> | **Revenue (last 24hs)** | Value of orders placed by users navigating in the version of your store associated to each workspace in the past 24 hours durint the test (according to the currency configured in the store).|

Reviewer 2: ```suggestion
| **Revenue (last 24hs)** | Total value of orders placed by users navigating the store version associated with each workspace in the past 24 hours during the test (expressed in the store's native currency).|
```
